swagger.json

```
@@ -2703,6 +2707,104 @@
           }
         }
       }
+    },
+    "/sct-treasury/deposit": {
+      "post": {
+        "summary": "/api/v1/sct-treasury/deposit",
```

For the reasons explained at app.js , the 
```suggestion is
        "summary": "/api/v1/scttreasury/deposit",
```